xgi/generators/uniform.py

```diff
@@ -277,7 +278,16 @@ def uniform_erdos_renyi_hypergraph(n, m, p, p_type="degree", seed=None):
         Mean expected degree if p_type="degree" and
         probability of an m-hyperedge if p_type="prob"
     p_type : str
-        "degree" or "prob", by default "degree"
+        "degree" or "prob", by default "prob"
```

Comment:
add something like "changes the way `p` is interpreted (see `p` for detail)

btw this option is not available in the other random_hypergraph() functions, should we make it?

Reply:
Fixed.